### PR TITLE
Update SessionStorage.php

### PR DESCRIPTION
**Pull request by asch3ron**
session_start NOTICE on crontab or if the session already start. (with last version of PHP)

We are using this library for our prod environnement using composer. It will be great if we didn't have the notice anymore ! Thanks for your work \o/

### CODE REVIEW

File: src/Happyr/LinkedIn/Storage/SessionStorage.php [hunk 1]
```diff
@@ -19,7 +19,11 @@ class SessionStorage implements DataStorageInterface
     public function __construct()
     {
         //start the session if it not already been started
-        @session_start();
+        if(php_sapi_name() != "cli") {
```
Comment (Nyholm) [inline src/Happyr/LinkedIn/Storage/SessionStorage.php]: Use a strict comparison. (!==)

File: src/Happyr/LinkedIn/Storage/SessionStorage.php [hunk 2]
```diff
@@ -19,7 +19,11 @@ class SessionStorage implements DataStorageInterface
...
+            if (!session_id()) {
```
Comment (Nyholm) [inline src/Happyr/LinkedIn/Storage/SessionStorage.php]: if (session_id() === "")

http://php.net/manual/en/function.session-id.php

**Comment by Nyholm** [comment]
Thank you for this PR. 
Every time you suppress an error with @ a kitten dies. Thank you for saving those kittens. =)

I wrote some minor comments that Sensiolabs insight is going to complain about if I merge. Please fix them and I'm happy to merge. 

Thank you.

File: src/Happyr/LinkedIn/Storage/SessionStorage.php (hunk 1 above)
Comment (asch3ron) [inline src/Happyr/LinkedIn/Storage/SessionStorage.php]: I just copy past the code in the Facebook SDK, but yes, some improvement to do :)

Comment (Nyholm) [inline src/Happyr/LinkedIn/Storage/SessionStorage.php]: Okey. This lib is much inspired by the Facebook SDK. Sadly that lib is not very good written. 

Thank you. =)

**Comment by Nyholm** [comment]
Pushed 0.3.1

**Pull request merged**